guides/Docker.md

```diff
@@ -62,6 +62,30 @@ Schedule the task below to run once a day.
 docker run -ti -e DATABASE_URL=... ankane/pghero bin/rake pghero:capture_space_stats
 ```
 
+## Historical Connection Stats
+
+To track space stats over time, create a table to store them.
+
+```sql
+CREATE TABLE "pghero_connection_stats" (
+  "id" bigserial primary key, 
+  "database" text, 
+  "ip" text, 
+  "source" text, 
+  "total_connections" bigint, 
```

Comment:
It looks like `ip` and `source` aren't used in capture, which makes sense, so let's remove them here. Same applies to other examples and migration.